.github/workflows/image.yml

```diff
@@ -0,0 +1,37 @@
+name: image
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
```

Comment:
what's the `workflow_dispatch:` for?

Reply:
that allows you to manually trigger the pipeline
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
cool 😎